packages/p2p/src/constants/user-recommendation.js

```diff
@@ -0,0 +1,4 @@
+export const user_recommendation = {
+    YES: 'yes',
+    NO: 'no',
+};
```

Comment:
can we use a boolean? 😮

Reply:
I had created this as a placeholder as API changes were yet to be merged. As per the API, the value returned is 1 or 0 instead of true or false. So I'll replace the string values with numerical values

Reply:
Yes returned value from BE is always 0 or 1, it has been implemented in the code previously and that's why we need to keep the code standard and use Boolean values the same way we use in other places.